packages/default-storage/windows/ReactNativeAsyncStorage/ReactNativeAsyncStorage.vcxproj

```diff
@@ -113,16 +131,24 @@
       <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions Condition="'$(UseFabric)'=='true'">USE_FABRIC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Shouldn't this be part of the props file? - Or do all apps have to do this?

Reply:
I believe this is a part of both the [cpp-app](https://github.com/microsoft/react-native-windows/blob/8a1386110bb75f7d54ec48c8ab97c289f10ba24f/vnext/templates/cpp-app/windows/MyApp/MyApp.vcxproj#L97) and the [cpp-lib](https://github.com/microsoft/react-native-windows/blob/8a1386110bb75f7d54ec48c8ab97c289f10ba24f/vnext/templates/cpp-lib/windows/MyLib/MyLib.vcxproj#L103) template, so it's done by all New Arch apps and libraries?

Reply:
Yes I plan on having this ready for 0.76.0, though it will probably be a new #define for customers (like RNW_NEW_ARCHITECTURE) : https://github.com/microsoft/react-native-windows/issues/13929